aws-servicebroker/README.md

```diff
@@ -80,72 +22,59 @@ name (or, alternatively, provide an `AWS_VPC` override to `make`).
       -passout pass:1234 \
       -subj '/CN=aws-servicebroker.${BROKER_NAMESPACE}' -batch \
       </dev/null
       ```
-    4. Decrypt the generated broker private key:
+    d. Decrypt the generated broker private key:
       ```
       openssl rsa -in tls.key.encrypted -passin pass:1234 -out tls.key
       ```
-    5. Sign the request with the CA certificate:
+    e. Sign the request with the CA certificate:
       ```
       openssl x509 -req -CA ca.pem -CAkey ca.key -CAcreateserial -in tls.req -out tls.pem
       ```
-4. Deploy the service broker:
-  Required substitutions:
-  - `BROKER_NAMESPACE`
-  - `AWS_ACCESS_KEY` (e.g. to output of `aws configure get aws_access_key_id`)
-  - `AWS_SECRET_KEY` (e.g. to output of `aws configure get aws_secret_access_key`)
-  - `AWS_REGION`
-  - `AWS_TARGET_ACCOUNT_ID` (to the account of the role to assume)
-  - `AWS_TARGET_ROLE_NAME` (to the role name of the role to assume)
-  - `AWS_VPC` (as created in the prerequisites)
-  ```
-  helm install \
-    --repo https://awsservicebroker.s3.amazonaws.com/charts \
-    aws-servicebroker \
-    --version 1.0.0-beta.3 \
-    --name aws \
-    --namespace ${BROKER_NAMESPACE} \
-    --set deployClusterServiceBroker=false \
-    --set authenticate=false \
-    --set aws.accesskeyid="${AWS_ACCESS_KEY}" \
-    --set aws.secretkey="${AWS_SECRET_KEY}" \
-    --set aws.region=${AWS_REGION} \
-    --set tls.cert="$$(base64 --wrap=0 tls.pem)" \
-    --set tls.key="$$(base64 --wrap=0 tls.key)" \
-    --set-string aws.targetaccountid=${AWS_TARGET_ACCOUNT_ID} \
-    --set aws.targetrolename=${AWS_TARGET_ROLE_NAME}} \
-    --set aws.vpcid=${AWS_VPC} \
-    --set brokerconfig.verbosity=0 \
-    --set image=awsservicebroker/aws-servicebroker:beta
-  ```
-5. Wait for things to become ready (by checking with `kubectl get pods --all-namespaces`)
-6. Create a service broker in CF:
+
+5.Install the AWS service broker as documented at 
+
+https://github.com/awslabs/aws-servicebroker/blob/master/docs/getting-started-k8s.md
+
+Skip the installation of the Kubernetes Service Catalog part. While installing the
+AWS service broker make sure to update the helm chart version (the version as of this
+writing is 1.0.0-beta.3). For the broker install, pass in a value indicating the Cluster
+Service Broker should not be installed.
+
+`helm install aws-sb/aws-servicebroker --name aws-servicebroker --namespace aws-sb --version 1.0.0-beta.3 --set aws.secretkey=$aws_access_key --set aws.accesskeyid=$aws_key_id --set deployClusterServiceBroker=false --set tls.cert="$(base64 -w0 tls.pem)" --set tls.key="$(base64 -w0 tls.key)" --set aws.tablename=awssb --set aws.vpcid=$vpcid --set aws.region=$aws_region --set authenticate=false`
+
+6.Create a service broker in CF:
   ```
-	cf create-service-broker aws unused-username unused-password  https://aws-servicebroker.${BROKER_NAMESPACE}
+	cf create-service-broker aws-servicebroker unused-username unused-password  https://aws-servicebroker.${BROKER_NAMESPACE}
```

Comment:
In the `helm install` command `${BROKER_NAMESPACE}` has been set to `aws-sb` whereas other commands still use the variable.